helm/charts/oxauth/values.yaml

```diff
@@ -0,0 +1,47 @@
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+dynamicStorage: true
+
+image:
+  repository: gluufederation/oxauth
+  tag: 4.0.0_01
+  pullPolicy: Always
+
+persistence:
+  size: 10M
+  accessModes: ReadWriteOnce
+  storageClassName: ""
+
+# VolumeClaim values
+volumeClaim:
+  storage: 50M
+
+#ConfigMap values
+gluuLdapUrl: opendj:1636
+gluuMaxRamFraction: 1
+
+#Resources for the deployment
+requests:
+  memory: 128Mi
```

Comment:
128 Mi ?

Reply:
Fixed. It was for the development purposes